.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -0,0 +1,36 @@
+name: Question
```

Comment:
Could you add the python version field as in the bug template and both that and the `issue-faced` required fields?